### DISCUSSION TITLE
NUE-ZBFLB (Nue Fan Light Switch) appearing as HGZB-4S (4 gang switch) in Zigbee2MQTT

### DISCUSSION THREAD

**Discussion by jasonc-glitch** [Koenkk/zigbee2mqtt#24043] [Devices]
I have a NUE-ZBFLB (Nue Fan Light Switch) installed and when I add it into Zigbee2MQTT it appears as a HGZB-4S (4 gang switch), not a fan/light switch.  It has four buttons exposed which work if I press them, but doesn't recognize that it is a fan controller where 3 of the switches are low/med/high for the fan.

Any ideas on how to get the fan/light switch to be identified correctly?  Note I have reset and re-joined it and the same issue occurs.

Thanks

**Comment by ajvandr** [comment]
I have the same thing.  Just bought 4 of them and they register as 4-Gang.
